src/vivarium_cluster_tools/psimulate/paths.py

```diff
@@ -72,12 +73,14 @@ def from_entry_point_args(
         command: str,
         input_artifact_path: Optional[Path],
         result_directory: Path,
+        input_model_spec_path: Path,
     ) -> "OutputPaths":
         launch_time = datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
 
         output_directory = result_directory
         if command == COMMANDS.run:
-            output_directory = output_directory / input_artifact_path.stem / launch_time
+            location = get_output_location_like_string(input_artifact_path, input_model_spec_path)
```

Comment:
Here too, there's no real reason to document that this is a location.  You can use this function (renamed to something more sensible like `get_model_name`) without making reference to the fact that it's a location.  E.g. PRL artifacts may not end up being named after locations, but after things like `census` and `cancer_registry` as we use vivarium to generate synthetic data for record linkage.